packages/expo-modules-core/common/cpp/JSIUtils.h

```diff
@@ -9,6 +9,13 @@ namespace jsi = facebook::jsi;
 
 namespace expo::common {
 
+#pragma mark - Helpers
+
+/**
+ Gets the core Expo object, i.e. `global.expo`.
+ */
+jsi::Object getCoreObject(jsi::Runtime &runtime);
```

Comment:
```suggestion
inline jsi::Object getCoreObject(jsi::Runtime &runtime);
```

nit: not sure whether inline helps here.

Reply:
I think it would be beneficial to add the `inline` modifier here. It might be ignored by the compiler, but that's okay.